OADP-5647 Independent BSL conflict (#89116)
openshift/openshift-docs: modules/oadp-configuring-dpa-multiple-bsl.adoc

[id="oadp-configuring-dpa-multiple-bsl_{context}"]
= Configuring the DPA with more than one BSL

> You can configure the `DataProtectionApplication` (DPA) custom resource (CR) with more than one `BackupStorageLocation` (BSL) and specify the credentials provided by the cloud provider.

Reviewer: ```suggestion
You can configure the `DataProtectionApplication` (DPA) custom resource (CR) with more than one `BackupStorageLocation` (BSL) CR and specify the credentials provided by the cloud provider.
```

Object name is needed: [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)

> 

Reviewer 2: Add a space before (CR)

Author: done. thanks.

[NOTE]
====

> Consider the following scenario in which you have configured two BSLs. You have:

Reviewer 3: ```suggestion
Consider the following scenario, where you have configured the following two BSLs:
```

> Consider the following scenario in which you have configured two BSLs. You have:

Reviewer 3: ```suggestion
Consider the following scenario, where you have configured the following two BSLs:
```

* Configured one BSL in the DPA and set it as the default BSL.
* Created another BSL independently by using the `BackupStorageLocation` CR.

In this scenario, as you have already set the BSL created through the DPA as the default, you cannot set the independently created BSL again as default.

> 

Reviewer 2: Consider "because" instead of "as".

Sounds like you're saying that creating a new BSL does not overwrite the existing default BSL. If that's the case, you might want to say this explicitly. 

This also might be important enough that you either use the IMPORTANT admonition or make it part of the text, instead of a note.  

All of the above are up to you since you know the product and user expectations better than I do.

Reviewer 3: I don't think it it used instead of "because".

Reviewer 3: +1. If it possible to remove "in this scenario" because it is already mentioned in the beginning of the admonition. How about the following statement -
Since you have already set the BSL created through the DPA as the default, you cannot set the independently created BSL again as default.

> [NOTE]
> ====
> Consider the following scenario, where you have configured the following two BSLs:
> 
> * Configured one BSL in the DPA and set it as the default BSL.
> * Created another BSL independently by using the `BackupStorageLocation` CR.
> 
> Because you have already set the BSL created through the DPA as the default, you cannot set the independently created BSL again as default.
> 
> This means, at any given time, you can set only one BSL as the default BSL.
> ====

Reviewer: ```suggestion
[NOTE]
====
You can set only one of the two BSLs as default at a time. For example, if you set a BSL that is configured in the DPA as default, do not set the other BSL as default, which is independently configured by using the `BackupStorageLocation` CR.
====
```

As per my understanding, the note guides about making the either of BSLs a default one. IMO, the note can be summed up as suggested above by reducing some words as per the minimalism principle 4.

> [NOTE]
> ====
> Consider the following scenario, where you have configured the following two BSLs:
> 
> * Configured one BSL in the DPA and set it as the default BSL.
> * Created another BSL independently by using the `BackupStorageLocation` CR.
> 
> Because you have already set the BSL created through the DPA as the default, you cannot set the independently created BSL again as default.
> 
> This means, at any given time, you can set only one BSL as the default BSL.
> ====

Reviewer 4: ```suggestion
For example, where you have configured the following two BSLs:

* Configured one BSL in the DPA and set it as the default BSL.
* Created another BSL independently by using the `BackupStorageLocation` CR.

As you have already set the BSL created through the DPA as the default, you cannot set the independently created BSL again as the default. This means, at any given time, you can set only one BSL as the default BSL.

```

.Procedure

> . Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`. See the following example.

Reviewer 3: ```suggestion
. Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`. See the following example:
```
Since there is an example title added, how about removing that part from the step description and having - Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`:

> . Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`. See the following example.

Reviewer 3: ```suggestion
. Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`. See the following example:
```
Since there is an example title added, how about removing that part from the step description and having - Configure the DataProtectionApplication with more than one BackupStorageLocation:

> . Configure the `DataProtectionApplication` with more than one `BackupStorageLocation`. See the following example:

Reviewer: ```suggestion
. Configure the `DataProtectionApplication` CR with more than one `BackupStorageLocation` CR. See the following example:
```

- Object name is needed: [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)

> An object reference is when you are referring to the actual instance of an API object, where the object name is important.
